homework/src/index.js

```diff
@@ -1,3 +1,6 @@
+
+
+/* global fetchJSON, createandAppend */
```

Comment:
This comment serves no purpose since `fetchJSON` and `createAndAppend` are defined in this same file. In VSCode these two variables are underlined, indicating an issue. You also misspelled `createAndAppend`.